### PR TITLE
Docu

### PR DESCRIPTION
**Pull request by calispac**
This is a start of the software documentation. I will start by documenting `digicapipe.io.containers` to describe the fields.

I also need this PR to include documentation build with Travis. 
This branch needs #60 to be merged.

### CODE REVIEW

**Comment by calispac** [comment]
I will try to use github pages to host the documentation.

**Comment by calispac** [comment]
I need this to be merged to try githubpages

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+SPHINXOPTS    =
+SPHINXBUILD   = python -msphinx
+SPHINXPROJ    = digicampipe
+SOURCEDIR     = docs
+BUILDDIR      = build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
```
Comment (vic0ward) [inline Makefile]: Add clean option to Makefile

File: baseline_stability.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                      default=os.path.realpath('digicampipe/tests/resources'))
+    parser.add_option('-f' , '--file', dest='filename', help='file basename e.g. CRAB_01_0_000.%03d.fits.fz',
+                      default='/example_100_evts.%03d.fits.fz', type=str)
+    # parser.add_option("-o", "--output", dest="output", help="output filename", default="output_crab.txt", type=str)
+    # parser.add_option("-d", "--display", dest="display", action="store_true", help="Display rather than output data",
+    #                   default=False)
+    parser.add_option('-s', "--file_start", dest='file_start', help='file number start', default=0, type=int)
+    parser.add_option('-e', "--file_end", dest='file_end', help='file number end', default=0, type=int)
+    parser.add_option('-c', "--camera_config", dest='camera_config_file', help='camera config file to load Camera()'
+                      , default=home_absolute_path + '/ctasoft/CTS/config/camera_config.cfg')
```
Comment (vic0ward) [inline baseline_stability.py]: Use config file from test ressource to avoid user dependent path

**Pull request merged**